TemplateManagement/PowerShell/ActiveDirectoryJoinV2/Join-Domain-StudentVM.ps1

```diff
@@ -70,7 +70,7 @@ try {
         exit
```

Comment:
We should use the full script name, "Join-Domain-StudentVM.ps1" and instead of stating what didn't happen, should list the requirement, "Join-Domain-StudentVM.ps1 script must be run on a student virtual machine in Lab Services"